lib/utils/export/to-zip.js

```diff
@@ -0,0 +1,21 @@
+import JSZip from 'jszip';
+
+export const noteExportToZip = notes => {
+	const zip = new JSZip();
+
+	notes
+		.activeNotes
+		.forEach(
+			( { id, content } ) => zip.file( `${ id }.txt`, content )
```

Comment:
> Also users have requested that the tags be in the exported file as well

I thought about adding them as hashtag strings. do you like `Tags: ToDo, Shopping` better than `#ToDo #Shopping`